core/src/main/scala/besom/internal/logging.scala

```diff
@@ -114,60 +114,107 @@ object logging:
     ): Result[Unit] =
       log(makeLogRecord(level, messages.toList, pkg, fileName, name, line, mdc), urn, streamId, ephemeral)
 
+    // TODO resource-less methods
+
+    def trace(message: LoggableMessage)(using
+      pkg: sourcecode.Pkg,
+      fileName: sourcecode.FileName,
+      name: sourcecode.Name,
+      line: sourcecode.Line,
+      mdc: BesomMDC[_]
+    ): Result[Unit] = log(Level.Trace, mdc, URN.empty, 0, false, message)
+
     def trace(message: LoggableMessage, res: Option[Resource] = None, streamId: Int = 0, ephemeral: Boolean = false)(using
       pkg: sourcecode.Pkg,
       fileName: sourcecode.FileName,
       name: sourcecode.Name,
       line: sourcecode.Line,
-      mdc: BesomMDC[_] = BesomMDC.empty
+      mdc: BesomMDC[_] = BesomMDC.empty,
```

Comment:
Now it won't be possible to call this method without `mdc`, unless one passes `ctx` as a named argument. This is a source incompatible change. Was this supposed to work like that now? `ctx` isn't used actually at the moment

Reply:
This is not a public API. This logger is purely internal, it's not even used by published packages.